examples/L2tracking/L2tracking_main.py

```diff
@@ -4,46 +4,45 @@
 from L2tracking_PDEconstraint import PoissonSolver
 from L2tracking_objective import L2trackingObjective
 
-#setup problem
+# setup problem
 mesh = fd.UnitSquareMesh(100, 100)
 Q = fs.FeControlSpace(mesh)
 inner = fs.ElasticityInnerProduct(Q)
 q = fs.ControlVector(Q, inner)
 
-#setup PDE constraint
+# setup PDE constraint
 mesh_m = Q.mesh_m
 e = PoissonSolver(mesh_m)
 
-#save state variable evolution in file u.pvd
+# save state variable evolution in file u.pvd
 e.solve()
 out = fd.File("u.pvd")
-cb = lambda: out.write(e.solution)
-cb()
 
-#create PDEconstrained objective functional
-J_ = L2trackingObjective(e, Q, cb=cb)
+# create PDEconstrained objective functional
+J_ = L2trackingObjective(e, Q, cb=lambda: out.write(e.solution))
```

Comment:
Before, we called `cb()` explicitely to save the initial domain. I suppose this is not necessary anymore. Please confirm

Reply:
turns out this was never needed in the first place, ROL calls `update` at the initial guess and then `cb` is called.